main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -56,6 +59,10 @@ public static void main(String[] argv) {
           "⚠️ Note that parameters marked with an asterisk (*) in the help menu are mandatory.");
       return;
     }
+    if (args.getExportNoticeSchema()) {
```

Comment:
I agree with [this comment](https://github.com/MobilityData/gtfs-validator/pull/925#discussion_r675050396), but this modification removes the ability to extract the schema when a dataset is being validated. Is this what we want? Could we do
```
if (args.getExportNoticeSchema()) {
    exportNoticeSchema(args);
    if(hasNoOtherArgs(args)) { 
        return; 
    }
}
```
or something like that?

Reply:
Makes sense to me! What do you think about adding a new CLI parameter `--abort_after_schema_extraction, -a` to represent this behavior? The default value would be set to `true`.

Reply:
edit: default value is set to `false` (see https://github.com/MobilityData/gtfs-validator/pull/925/commits/6e97a97e038720fb700bcb17337746bcbacd02c5)

Reply:
IMHO adding yet another command-line parameter for this behavior is confusing. As a user, I would expect that if I provide the parameter to dump the schema it dumps the schema, if I provide the validation parameters (in particular `input`) then it validates the dataset, and if I provide both it does both.

Reply:
Changed in https://github.com/MobilityData/gtfs-validator/pull/925/commits/3f591e2725b64f412907c4e2df7a6c05b9439036.